app/views/template/Layout.scala.html

```diff
@@ -0,0 +1,91 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.hmrcfrontend.views.html.helpers._
+@import uk.gov.hmrc.hmrcfrontend.views.config.StandardBetaBanner
+@import uk.gov.hmrc.hmrcfrontend.views.html.helpers.HmrcReportTechnicalIssueHelper
+@import views.html.helper.CSPNonce
+@import config.FrontendAppConfig
+@import controllers.routes
+
+@this(
+appConfig: FrontendAppConfig,
+govukLayout: GovukLayout,
+hmrcHead: HmrcHead,
+hmrcTimeoutDialogHelper: HmrcTimeoutDialogHelper,
+hmrcStandardFooter: HmrcStandardFooter,
+hmrcStandardHeader: HmrcStandardHeader,
+standardBetaBanner: StandardBetaBanner,
+defaultMainContentLayout: TwoThirdsMainContent,
+hmrcScripts: HmrcScripts,
+govukBackLink: GovukBackLink,
+hmrcReportTechnicalIssueHelper: HmrcReportTechnicalIssueHelper
+)
+
+@(pageTitle: String,
+timeoutEnabled: Boolean = true,
+canSignOut: Boolean = true,
+showBackLink: Boolean = true,
+fullWidthLayout: Boolean = false
+)(mainContent: Html)(implicit request: Request[_], messages: Messages)
+
+@headScripts = {
+@if(timeoutEnabled) {
+@hmrcTimeoutDialogHelper(
```

Comment:
I left a previous comment about this but it may have been misunderstood. We have two values in `FrontendAppConfig` that are to be used for the timeout and countdown values for this component. Can you please use those here (check AMLS AMP service if you are still unsure).